plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -63,8 +63,10 @@
 import static com.google.common.collect.ImmutableList.toImmutableList;
```

Comment:
Typos in commit message:

ParquetSchemaConvert**o**r -> ParquetSchemaConverter 
inten**e**ded -> intended